ds-pipelines-targets-step5.R

```diff
@@ -0,0 +1,44 @@
+# Libraries
+
+    library(dplyr)
+    library(readr)
+    library(stringr)
+    library(sbtools)
+    library(whisker)
+
+# Paths and filenames
+  
+    fetch_output_dir <- 'fetch/out/' # fetch output directory
+    process_output_dir <- 'process/out/' # process output directory
+    fetch_src_dir <- 'fetch/src/'
+    process_src_dir <- 'process/src/'
+    
+    model_data <- paste(fetch_output_dir, 'model_RMSEs.csv') # model RMSE output file
```

Comment:
I like where you are going with this - just having the one spot for directories, but it actually makes it harder to read because when I look down at the filepaths below, I have to keep looking up to see what the value of the `dir` strings are. I think this is one time where shared objects are actually causing it to be less legible. I would suggest leaving it simpler like this:

```r
source('fetch/src/1_getData.R')
source('process/src/2_prepData.R')
source('process/src/3_plotData.R')
source('process/src/4_saveDiagnostics.R')
```

I really like how you are using numbering to organize. I think we want is for the numbering to happen at the top-level to define and order the `fetch`, `process` and `visualize` phases, so I would change those to `1_fetch`, `2_process`, and `3_visualize`. So your source call would become:

```r
source('1_fetch/src/getData.R')
source('2_process/src/prepData.R')
source('3_visualize/src/plotData.R')
source('2_process/src/saveDiagnostics.R')
```